sonar-plugin-api/src/main/java/org/sonar/api/measures/Metric.java

```diff
@@ -231,7 +231,10 @@ public Integer getId() {
 
   /**
    * @return the metric formula
+   * @deprecated since 5.2 there's no more decorator on batch side
+   * TODO add link to no API
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)